client/App.jsx

```diff
@@ -1,15 +1,23 @@
 import React from 'react';
 import { Routes, Route } from 'react-router-dom';
 import './styles/styles.scss';
-
+import ThemeContext from './components/ThemeContext';
+import { Home } from './pages/Home'
 // import { Login } from './pages/Login';
-import {Home} from './pages/Home'
+import { useState, useEffect } from 'react';
 
 export function App() {
+
+  //& Using ThemeContext to have 'username' and 'setUserName' as global variables throughout are app (ITS OP!)
+  const [username, setUserName] = useState('currymonstanacho')
+
   return (
-    <Routes>
-      {/* <Route path="/" element={<Login />} /> */}
-      <Route path = '/home' element = {<Home/>}/>
-    </Routes>
+    <ThemeContext.Provider value={{username, setUserName}}>
```

Comment:
Change the context to be called UserContext.Provider to match proper naming convention